concepts/pointers/about.md

```diff
@@ -0,0 +1,404 @@
+# About
+
+A pointer is a variable or constant whose value is the address of another variable or constant.
+Pointers are a powerful and potentially dangerous feature of C.
+It can be challenging to understand pointers.
+
+```c
+int i = 5;
+```
+
+When we assign a value to a variable we can then ask two questions.
+
+Who are you?
+
+_I am `i`_.
+
+What do you have?
+
+_I have the integer `5`_.
+
+There is also a third question we can ask.
+
+Where are you?
+
+_I am at `0x6dfed4`_.
+
+Wait, what?
+
+Variables and constants store their value in a memory location.
+That location in memory is known as the address of the variable or constant.
+
```

Comment:
Worth mentioning that quite often addresses are displayed in hexadecimal format by convention?
Just a suggestion, not sure if this is needed or not.